{
  EventRecord.here(CallbackListenerSequence.class,EventType.CALLBACK,"fire(" + response.getClass().getName() + ")").debug();
  for (  Callback<R> cb : this.successCallbacks) {
    try {
      EventRecord.here(cb.getClass(),EventType.CALLBACK,"fire(" + response.getClass().getName() + ")").debug();
      cb.fire(response);
    }
 catch (    Throwable t) {
      LOG.error("Exception occurred while trying to call: " + cb.getClass().getSimpleName() + ".apply( "+ t.getMessage()+ " )");
      LOG.error(t,t);
    }
  }
}
